src/v2/ion/i18nTransformer.js

```diff
@@ -187,7 +217,8 @@ const updateLabelForUiSchema = (remediation, uiSchema) => {
 
   if (uiSchema.label) {
     Logger.info('\t 2: ', i18nPath);
-    uiSchema.label = getI18NValue(i18nPath, uiSchema.label);
+    const params = getI18NParams(remediation, authenticatorKey);
+    uiSchema.label = getI18NValue(i18nPath, uiSchema.label, params);
```

Comment:
Suggestion: Instead of requiring a call to `getI18NParams` here, let's nest it within the `getI18nValue` method. That way, all our i18n lookup logic is centralized.

Reply:
I looked into this, and it will affect all uiSchemas then. I thought we should restrict this to labels. What do you think?

Reply:
I'd assume we'll also want this for additional schemas, but am fine if we keep this here for now!